File: library-db/src/main/java/com/my/library_db/db/AppDatabase.java
package com.my.library_db.db;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.my.library_db.dao.UserDao;
import com.my.library_db.model.User;

@Database(entities = {User.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract UserDao userDao();
}

File: library-db/src/main/java/com/my/library_db/db/AppDatabase.kt
package com.my.library_db.db

import androidx.room.Database
import androidx.room.RoomDatabase
import com.my.library_db.dao.UserDao
import com.my.library_db.model.User

@Database(entities = [User::class], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun userDao(): UserDao
}